src/common/interfaces.ts

```diff
@@ -5,30 +5,13 @@ export interface IConfig {
   has: (setting: string) => boolean;
 }
 
-export interface IExporterTaskParameters {
-  dbId: string;
-  crs: string;
-  zoomLevel: number;
-  callbackURL: string;
-  bbox: unknown;
-  expirationTime: Date;
-  priority: number;
-  tilesPath: string;
-  footprint: unknown;
-  packageName: string;
+export interface ICleanupData {
+  directoryPath?: string;
+  cleanupExpirationTimeUTC?: Date;
 }
 
-export interface IExporterJobParameters {
-  userId: string;
-  footprint: unknown;
-  bbox: unknown;
-  version: string;
-  cswProductId: string;
-  tilesPath: string;
-  priority: number;
-  crs: string;
-  fileName: string;
-  relativeDirectoryPath: string;
+export interface IExporterCleanupParameters {
+  cleanupData?: ICleanupData;
 }
 
 export interface ITaskResponse<T> {
```

Comment:
is this interface needed? "ITaskResponse" - can it be taken from mc-priority-queue?

Reply:
it wad removed on previoues pr as we talk. that the task parameters are generic and not relevant specificlly on cleanup that looking for job's parameters only to find cleanupData object